qa/system-indices/src/main/java/org/elasticsearch/system/indices/SystemIndicesQA.java

```diff
@@ -110,7 +110,7 @@ private static XContentBuilder mappings() {
             return jsonBuilder().startObject()
                 .startObject(SINGLE_MAPPING_NAME)
                 .startObject("_meta")
-                .field("version", Version.CURRENT)
+                .field("version", Build.current().version())
```

Comment:
My understanding of the way we are handling `_meta` is to introduce a new version **in parallel** (`_meta.managed_index_mappings_version` - from `SystemIndexDescriptor.VERSION_META_KEY`). I see this was done for logstash, watch, and the various ML responses etc.

I think the plan is to transition to the new one, and then remove all the old `_meta.version` stuff.
My understanding is that this cannot be done now as there is some code path (now deprecated) that still tries to extract a Version ( `extractNodeVersionFromMappings`), and that is needed for older 8.x versions.

For the purposes of this test it probably does not matter though, and can be anything (or probably even removed?) We may leave this as-is, and when we remove all `_meta.version` it will fall down.

Reply:
I think you can hard-code `8.11.0`. I know ML is using that as a placeholder so that pre-8.11 nodes will parse the mapping as set by a future node.

If tests pass when you remove it, that's fine too.

Reply:
Great! I will try to just remove it; if security it's not involved it should pass. Let me try it.